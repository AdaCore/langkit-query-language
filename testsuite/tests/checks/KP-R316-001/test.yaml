driver: 'checker'
rule_name: KP_R316_001
project: 'prj.gpr'
