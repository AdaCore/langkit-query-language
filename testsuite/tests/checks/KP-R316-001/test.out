p.ads:9:13: rule violation: possible occurrence of KP R316-001
9 |    function Func_Lim_Array (Int : Integer) return Lim_Array;  --  FLAG
  |             ^^^^^^^^^^^^^^

